src/nisarqa/input_product_readers/radar_product.py

```diff
@@ -206,12 +206,24 @@ def _get_raster_from_path(
         )
         kwargs["zero_doppler_time"] = h5_file[path][...]
 
+        # From the xml Product Spec, zeroDopplerTimeSpacing is the
+        # '...spacing between consecutive entries in the zeroDopplerTime array'.
+
+        # However, this is a slight misnomer. "Spacing" refers to the
+        # (positive-valued) width of a pixel while "posting" refers to
+        # the (positive- or negative-valued) stride between points in a grid.
+        # (The spacing is the absolute value of the posting.)
+
+        # For NISAR L1 products, `zeroDopplerTimeSpacing` is actually the
+        # y-coordinate posting. Note that for NISAR L1 products (i.e.
+        # radar grids), the y-coordinate posting is positive
+        # (the positive y-axis points down in QA plots).
```

Comment:
As discussed in a previous comment, it's common to just use "spacing" (not "posting") when referring to the radar grid. As such, we could revert the changes to this `radar_product.py` module.